tripServer/server.js

```diff
@@ -1 +1,16 @@
 const express = require('express');
+// const morgan = require('morgan');
+// const config = require('./config/maps');
+const requestHandler = require('./requestHandlers.js').requestHandler;
+
+const app = express();
+
+app.get('/api/trip', requestHandler);
```

Comment:
I think the agreed endpoint for the trip server is just '/trip'; 'api/trip' will be going to the Integration server